src/operationDefinitions/types.ts

```diff
@@ -5,20 +5,36 @@
 
 import { Router } from 'express';
 import { SystemOperation, TypeOperation } from 'fhir-works-on-aws-interface';
+import ResourceHandler from '../router/handlers/resourceHandler';
 
 export interface OperationDefinitionImplementation {
     /**
      * url of the corresponding OperationDefinition resource
      */
-    canonicalUrl: string;
+    readonly canonicalUrl: string;
+
+    /**
+     * url path used to invoke the operation
```

Comment:
I think adding an example here will be helpful. Like do we expect the '/' to be there in the start or no?

```suggestion
     * url path used to invoke the operation. Example: "/$docref"
```

Reply:
added the example